x/recovery/keeper/ibc_callbacks_integration_test.go

```diff
@@ -103,7 +103,7 @@ var _ = Describe("Recovery: Performing an IBC Transfer", Ordered, func() {
 						s.EvmosChain.App.(*app.Evmos).ClaimsKeeper.SetClaimsRecord(s.EvmosChain.GetContext(), senderAcc, claim)
 
 						// update the escrowed account balance to maintain the invariant
-						err := testutil.FundModuleAccount(s.EvmosChain.App.(*app.Evmos).BankKeeper, s.EvmosChain.GetContext(), claimtypes.ModuleName, coins)
+						err := testutil.FundModuleAccount(s.EvmosChain.GetContext(), s.EvmosChain.App.(*app.Evmos).BankKeeper, claimtypes.ModuleName, coins)
```

Comment:
nit, this change is unrelated to the context of this PR. Consider opening a new one for it